models/Recipe.model.js

```diff
@@ -1,10 +1,50 @@
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
 
 const recipeSchema = new Schema({
   // TODO: write the schema
+  title: {
+    type: String,
+    required: true,
+    uniqued: true,
+  },
+  level: {
+    type: [String],
+    enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"],
+  },
+
+  ingredients: [String],
+  cuisine: {
+    type: String,
+    requiered: true,
+  },
+  dishType: {
+    type: [String],
+    enum: [
+      "breakfast",
+      "main_course",
+      "soup",
+      "snack",
+      "drink",
+      "dessert",
+      "other",
+    ],
+  },
+  image: {
+    type: String,
+    default: "https://images.media-allrecipes.com/images/75131.jpg",
+  },
+  duration: {
+    type: Number,
+    minimum: 0,
```

Comment:
The correct mongoose validator here would be just "min"! For more info on them, check [https://mongoosejs.com/docs/validation.html](url). 👋